ansible-st2-local/roles/arteria/tasks/main.yml

```diff
@@ -6,8 +6,15 @@
 - include: conf.yml
   tags: arteria_master_conf
 
-- name: Pause for 1 minute until stackstorm services are up and online
-  pause: minutes=1
+# This is needed because handlers are run last, after all tasks are finished, 
```

Comment:
Perhaps we should rather use `- meta: flush_handlers` here? That will force all handlers to be executed.